timemachine/fe/atom_mapping.py

```diff
@@ -451,8 +430,9 @@ def leaf_filter_fxn(trial_core) -> bool:
 
         # distance score
         r2_ij = np.sum(np.power(r_i - r_j, 2))
-        rmsd = np.sqrt(r2_ij / len(core))
-        dists.append(rmsd)
+        # No need to perform square root here, only care about ordering
+        mean_sq_dist = r2_ij / len(core)
```

Comment:
A negligible difference, but no reason to take the square root since we only use this value to order.